src/main/kotlin/org/rust/lang/core/completion/RsImplTraitMemberCompletionProvider.kt

```diff
@@ -128,22 +108,18 @@ private fun completeType(target: RsTypeAlias): LookupElementBuilder {
 private fun completeFunction(
     target: RsFunction,
     impl: RsImplItem,
-    substitution: RsPsiSubstitution
+    substitution: RsPsiSubstitution,
+    memberGenerator: MembersGenerator
 ): LookupElementBuilder {
-    val renderer = ImportingPsiRenderer(PsiRenderingOptions(renderGenericsAndWhere = true), substitution, impl)
     val shortRenderer = ImportingPsiRenderer(PsiRenderingOptions(renderGenericsAndWhere = false), substitution, impl)
```

Comment:
By the way, `ImportingPsiRenderer` looks too heavy for the completion presentation. Maybe it's better to use simple `PsiSubstitutingPsiRenderer` (with default `PsiRenderingOptions.shortPaths = true` option)?

Reply:
Good point, using the importing renderer in presentation is probably not a good idea. I changed it per your suggestion.